keyboards/levinson/keymaps/atreus/keymap.c

```diff
@@ -0,0 +1,62 @@
+#include "levinson.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+#define _QW 0
+#define _RS 1
+#define _LW 2
+
+/*
+ *  q       w      e     r    t        ||       y     u     i     o    p
+ *  a       s      d     f    g        ||       h     j     k     l    ;
+ *  z       x      c     v    b        ||       n     m     ,     .    /
+ * esc     tab    gui  shift bksp ctrl || alt space raise   -     '  enter
+ */
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[_QW] = KEYMAP( /* Qwerty */
```

Comment:
`KEYMAP` is no longer used for this keyboard

Please replace this with `LAYOUT` or `LAYOUT_ortho_4x12`

```suggestion
[_QW] = LAYOUT( /* Qwerty */
```

```suggestion
[_QW] = LAYOUT_ortho_4x12( /* Qwerty */
```